app/views/events/index.html.erb

```diff
@@ -17,9 +17,9 @@
 		<li> <%= link_to "Edit", edit_event_path(event) %> </li>
 	<% end %>
 <% end %>
-</div>
+</div> 
```

Comment:
Very minor but there is a trailing whitespace perhaps added by accident
```suggestion
</div>
```

Reply:
:o how could you notice such thing lol

Reply:
Just somehow noticed it when going through the diff :) Github coloured the the extra whitespace nicely haha